README.md

```diff
@@ -2,31 +2,559 @@
+
+
+Since Java defines a `String` as a single unit of a sequence of characters, we
+can look at `String` objects as having indexes like we do with character arrays.
+For example, consider we have a `String` object with the value of "Cat". In
+Java, the 'C' would be at index 0, the 'a' would be at index 1, and the 't'
+would be at index 2. 
+
+![string charat](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/string_charat.png)
+
+
+Knowing this will help us manipulate our `String` objects
+more efficiently. There is even a `charAt()` method to help us get a character
+at a specific index within a `String`:
+
+```java
+public class StringCharAtExample {
+
+  public static void main(String[] args) {
+    String cat = "Cat";
+    char character = cat.charAt(1);
+
+    System.out.println(character);
+  }
+}
+```
+
+The following output of this would be:
+
+```text
+a
+```
+
+Recall from the previous lesson we iterated over the characters in a string
+by converting the string to an array of characters using the `toCharArray()`
+method:
+
+```java
+public static boolean containsA(String str) {
+    for (char c : str.toCharArray()) {
+        if (c == 'A') {
+            return true;
+        }
+    }
+    return false;
+}
+```
+
+We can avoid the need to convert to a character array by using the `charAt(int)` method.
+The loop uses positional indexing to obtain each character in the string:
+
+```java
+public static boolean containsA(String str) {
+    for (int i = 0; i < str.length(); i++) {
+        if (str.charAt(i) == 'A') {
+            return true;
+        }
+    }
+    return false;
+}
+```
+
+### Contains 
+
+| String Method                           | Description                                                                             |
+|-----------------------------------------|-----------------------------------------------------------------------------------------|
+| public boolean contains(CharSequence s) | Returns true if and only if this string contains the specified sequence of char values. |
+
+
+Consider this: We want to know if a `String` contains a specific sequence of
+characters. Maybe we are looking for the word `cat` within a string. We can
+confirm if the sequence of characters is contained within a `String` by calling
+the `contains()` method. The `contains()` method returns a boolean value - so if
+the sequence is found, it will return true, else it will return false.
+
+```java
+public class StringContainsExample {
+    public static void main(String[] args) {
+        String greeting = "Hello World";
+
+        boolean containsCat = greeting.contains("cat");
+        if (containsCat) {
+            System.out.println("We found the cat!");
+        } else {
+            System.out.println("I guess we need to keep looking!");
+        }
+    }
+}
+```
+
+![string contains](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-strings/string_contains.png)
```

Comment:
It looks like this image here has `greetings` instead of `greeting`. It might easier to just change the code above and some of the paragraph below instead of changing the image.

Reply:
thanks, updated the code and text.